### DISCUSSION TITLE
Authentication in download file URLs

### DISCUSSION THREAD

**Discussion by thomaskamalakis** [Budibase/budibase#5960] [Ideas]
Hi budibase team!

Maybe you could consider having some sort of authentication flow before accessing file attachments, i.e. when some one accesses

https://mybudibase.example.com/prod-budi-app-assets/{{myappid}}/attachments/{{myfileid}}

it will requires login if the user is not authenticated.

**Comment by melohagan** [comment]
Definitely!

I think this is related: https://github.com/Budibase/budibase/issues/3582
